k8s/cluster.yaml

```diff
@@ -139,7 +139,7 @@ spec:
       net.ipv4.neigh.default.gc_thresh2 = 4096
       net.ipv4.neigh.default.gc_thresh3 = 32768
       EOT
-  image: kope.io/k8s-1.17-debian-stretch-amd64-hvm-ebs-2020-01-17
+  image: testground_2020-06-08
```

Comment:
For now we don't have to - we can create them on-demand, when there is a new version of k8s, or when there is a new version of a dependency that we care about.

I suggested doing it on release, in case we want to include the `sidecar` and the `daemon`, but they are very small parts of all of the other images, so I wouldn't say this is a must.